djangoproject/djangoapp/views.py

```diff
@@ -1,12 +1,18 @@
 from django.shortcuts import render
-from rest_framework.decorators import api_view, permission_classes
-from rest_framework.permissions import IsAuthenticated
-from rest_framework.response import Response
-from .serializers import UserSerializer
 from django.contrib.auth.models import User
+from .serializers import UserSerializer
+from rest_framework import permissions, response, status, views, viewsets, mixins
+
+class CurrentUserView(views.APIView):
+
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get(self, request):          
+        response_serializer = UserSerializer(request.user)
+        return response.Response(response_serializer.data, status.HTTP_200_OK)
```

Comment:
ovo je okej, medjutim hajde onda i ovo dole da ubacimo da onda idemo sve preko jedne klase, ono sto mozes korsiti kako bi to postigao jeste da dole na nivou klase defisnises custom rutu , a evo kako https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing

Reply:
Pogledacu kako se to radi, pa cu napraviti preko jedne klase